### PR TITLE
Feature/participations

### PR DESCRIPTION
**Pull request by Fujistone**
Link to request : https://metabase.osp.cat/model/1429-participations/query

### CODE REVIEW

File: participations/locales/en.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+name: Participations
+description: >
+  All participations made on the platform : proposals, endorsements, proposals' votes, debates, answers to surveys and comments.
```
Comment (virgile-dev) [inline participations/locales/en.yml:4]: Add nuance about unregistered user survey answers. 
Add them to the mix using the session id creates potential duplicates that we can't verify.

File: participations/info.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
+query:
+  model: true
+  sql: |
+    -- /!\ Asked by city : not taking into account comments, endorsments and proposal votes in contributions !
```
Comment (virgile-dev) [inline participations/info.yml:5]: remove comment

Comment (Dynnammo) [reply participations/info.yml]: Fixed in [db95ca7](https://github.com/OpenSourcePolitics/decidim_cards/pull/42/commits/db95ca79aa1e2275149b39294497cb23cebe7dbf)

File: participations/locales/en.yml (hunk 1 above)
Comment (Dynnammo) [reply participations/locales/en.yml:4]: Fixed in 3ac05d8

File: participations/info.yml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        participation_date
+    from participations
+  info:
+    meta:
+      depends_on: 
+      - proposals
+      - endorsmements
+      - forms_answers
+      - debates
+      - projects_votes
```
[the remark this answers is not in the document]
Comment (Dynnammo) [reply participations/info.yml:87]: Fixed in 1b694a6

**Comment by Dynnammo** [comment]
Link to request : [metabase.osp.cat/model/1429-participations/query](https://metabase.osp.cat/model/1429-participations/query)

File: participations/locales/en.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+name: Participations
+description: >
+  All participations made on the platform : proposals, endorsements, proposals' votes, debates, answers to surveys and comments. 
+  Please note that it also counts answers of unregistered users, in this case they're identified by their session_id.
```
Comment (virgile-dev) [inline participations/locales/en.yml:5]: add ⚠️  before "Please ..."

**Pull request merged**